src/app/components/option-item/option-item.tsx

```diff
@@ -7,17 +9,21 @@ export type OptionItemProps = {
   href?: string;
 };
 
-export const OptionItem = ({ title, description, buttonLabel, href }: OptionItemProps) => (
+export const OptionItem = ({ title, description, buttonLabel, href = '' }: OptionItemProps) => (
   <div key={title} className="option">
     <h3 className="option-title">{title}</h3>
     <p className="option-description" role="text">
       {description}
     </p>
     {buttonLabel && (
-      <a href={href} target="_blank" rel="noreferrer" className="option-button">
-        {buttonLabel}
-        <span className="material-symbols-outlined arrow">arrow_forward</span>
-      </a>
+      <LinkCustom
+        label={buttonLabel}
```

Comment:
I suppose it will be better to use `linkLabel` as we use `Link` component here

Reply:
Changed